view/frontend/web/js/cmp.js

```diff
@@ -44,34 +44,82 @@ define(['jquery'], function ($) {
         } else if (cid === "") {
             console.warn("Nosto cid is missing. Segment mapping cannot be fetched")
             return;
-        } else if (config.cookieName === "") {
-            console.warn("Nosto mapping cookie name is missing. Segment mapping cannot be fetched")
+        } else if (config.categoryCookie === "") {
```

Comment:
Where is config provided?

Reply:
Where does it come from?

Reply:
From the template file https://github.com/Nosto/nosto-magento2-cmp/blob/d8e04b2473a5d16614bb51e9ca6058b60b58e26b/view/frontend/templates/category_merchandising.phtml#L45